.gitignore

```diff
@@ -4,5 +4,6 @@ $ cat .gitignore
 /dist
 /example-app/dist
 /coverage
-
+.idea
+.vscode
```

Comment:
Sure! (not sure why you whould want our IDE artifacts in there though ;)

Reply:
Well, the `settings.json` from VSCode is like a team-shared settings file. This is useful for consistent formatting etc. With VSCode you have your personal settings and then the workspace settings (which should be part of the repository). It's similar with other tools like Resharper in the .NET world where you also have multiple layers of settings (whole computer, solution for team, solution personal etc.) some of which are usually checked in, some of which are not.

Since I am not really an IntelliJ user I am not familiar with what files live in the `.idea` folder and whether it should be ignored or not. My comment regarding removing the line was really only meant for the `.vscode` folder.